### ISSUE TITLE
empty( self::$x ) reports 5.5.0

### ISSUE THREAD

**Issue by tivnet** [llaville/php-compat-info#238]
This code runs OK under 5.2, I believe. But it's reported as "5.5.0"

```php
<?php

class aa {
    public static $x = "a";
    public static function check(){
        if( !empty(self::$x)) echo "ok";
    }
}

aa::check();

**Comment by slaFFik** [comment]
Example check: https://3v4l.org/V02RW

**Comment by llaville** [comment]
Fixed in repo, bugfixe release 5.0.9 is on way !
Thanks for reporting.

**Comment by tivnet** [comment]
@llaville Works OK now, thank you!
